src/scss/layout/_layout.scss

```diff
@@ -57,35 +43,73 @@ article {
 			display: table;
 		}
 	}
-}
 
-/*
- * Default WP Alignment Classes
+	/*
+	WP selector's customized styles for the WeCount website
  *****************************************************************************/
-figcaption {
-	display: block;
-	font-size: rem(14);
-	font-style: italic;
-	padding: 1rem 0;
-	text-align: center;
+	figure {
+		display: table;
+		margin-left: auto;
+		margin-right: auto;
+	}
+
+	.wp-block-image {
+		img {
+			height: 100%;
+			max-width: 100%;
+		}
+
+		figcaption {
+			caption-side: bottom;
+			display: table-caption;
+			font-size: rem(14);
+			font-style: italic;
+			padding: 1rem 0;
+			text-align: center;
+		}
+
+		.alignleft {
+			float: left;
+			margin-right: rem(25);
+			margin-top: 0;
+		}
+
+		.alignright {
+			float: right;
+			margin-left: rem(25);
+			margin-top: 0;
+		}
+	}
+
+	.wp-block-image figure .alignfull {
```

Comment:
If you inspect DOM structures when `alignfull` or `alignwide` are applied, correct selectors for them are `.wp-block-image.alignfull` and `.wp-block-image.alignwide`. With wrong selectors here, the css from line 84-98 has never been applied. These 2 cases are actually using the default `figure` css at line 50. Please investigate and fix. Thanks.

Reply:
Tried it out and you are right. Looks like there may not be a need for the selectors on line 84-98.

Reply:
I agree.

Reply:
Okay, I will remove them.